components/tag-list/tag-list.js

```diff
@@ -314,37 +327,37 @@ class TagList extends LocalizeCoreElement(ArrowKeysMixin(LitElement)) {
 		}
 	}
 
-	_handleResize(entries) {
+	async _handleResize(entries) {
 		this._availableWidth = Math.floor(entries[0].contentRect.width);
 		if (this._availableWidth >= PAGE_SIZE.large) this._lines = PAGE_SIZE_LINES.large;
 		else if (this._availableWidth < PAGE_SIZE.large && this._availableWidth >= PAGE_SIZE.medium) this._lines = PAGE_SIZE_LINES.medium;
 		else this._lines = PAGE_SIZE_LINES.small;
 		if (!this._hasResized) {
 			this._hasResized = true;
-			this._handleSlotChange();
+			await this._handleSlotChange();
 		} else {
 			this._chomp();
 		}
 	}
 
-	_handleSlotChange() {
+	async _handleSlotChange() {
 		if (!this._hasResized) return;
+		this._contentReady = false;
+		await this.updateComplete;
 
-		setTimeout(async() => {
-			this._items = await this._getTagListItems();
-			if (!this._items || this._items.length === 0) {
-				this._chompIndex = 10000;
-				return;
-			}
+		this._items = await this._getTagListItems();
+		if (!this._items || this._items.length === 0) {
+			this._chompIndex = 10000;
+			return;
+		}
 
-			this._itemLayouts = this._getItemLayouts(this._items);
```

Comment:
I think without the `setTimeout`, `_getItemLayouts()` seems to be getting incorrect widths.  I log the `itemWidth` in the `_chomp()` method and compare that to what the browser console says.  The browser seems to need some time to do layout... not sure why...?

Reply:
Interesting - I'm getting the correct widths in `_chomp`. Is this just on Safari or also on Chrome?

Reply:
Hmmm. I am using Chrome right now.

Reply:
I updated my Chrome and now I get proper dimensions.  Interestingly, Safari is also getting proper values now.  So strange.  🤯